pkg/scheduler/apis/config/testing/policy_test.go

```diff
@@ -36,7 +36,7 @@ extenders:
   filterVerb: filter
   prioritizeVerb: prioritize
   weight: 1
-  enableHttps: false
+  enableHttps: true
```

Comment:
why did this change?

Reply:
If you make a typo in the test here, e.g. call it `ENABLEhttps` the test would still pass. Basically, I'm avoiding a default Go value for a non-nil type here.